internal/testhelpers/e2e/session.go

```diff
@@ -207,7 +207,10 @@ func new(t *testing.T, retainDirs, updatePath bool, extraEnv ...string) *Session
 	session.Exe = session.copyExeToBinDir(exe)
 	session.SvcExe = session.copyExeToBinDir(svcExe)
 	session.TrayExe = session.copyExeToBinDir(trayExe)
-	session.InstallerExe = session.CopyExeToDir(installExe, dirs.Base)
+	session.InstallerExe = session.CopyExeToDir(installExe, dirs.InstallerBin)
+	session.CopyExeToDir(exe, filepath.Join(dirs.InstallerPayload, installation.BinDirName))
+	session.CopyExeToDir(svcExe, filepath.Join(dirs.InstallerPayload, installation.BinDirName))
+	session.CopyExeToDir(trayExe, filepath.Join(dirs.InstallerPayload, installation.BinDirName))
```

Comment:
This creates extra IO for every single integration test. Considering we only need it in two tests can we move this logic into the test itself?